### ISSUE TITLE
hGetContents: invalid argument (invalid byte sequence)

### ISSUE THREAD

**Issue by nh2** [ndmitchell/hlint#96]
Running hlint v1.9.11 on my file containing unicode characters, I get:

```
hlint: myfile.hs: hGetContents: invalid argument (invalid byte sequence)
```

The problem is explained by @hvr here: https://github.com/finnsson/template-helper/issues/2

**Comment by xor-xor** [comment]
I'm still getting this error with hlint 1.9.21 and 1.9.20 (1.9.19 works fine, though).

**Comment by ndmitchell** [comment]
@xor-xor I'm unaware of any changes that happened with 1.9.20. Can you tell me exactly what you are running, what the error is etc.

**Comment by xor-xor** [comment]
It's weird... I can't reproduce this problem today. Maybe because I've made some changes to the file that I was hlinting (and yes, I forgot to make a copy of it when I encountered this problem - but I think that there was nothing special about this file). The error message was:
```
hlint: myfile.hs: hGetContents: invalid argument (invalid byte sequence)
```
(I'm on OS X 10.9.5 w/ Haskell Platform 2014.2.0.0; hlint installed in a sandbox)

If I manage to reproduce that somehow, I'll let you know.

Sorry for the noise! :wink:
